src/components/WarningModal.jsx

```diff
@@ -28,21 +25,23 @@ const WarningModal = ({
       </div>
       {!noButtons && (
         <div {...classes('buttons')}>
-          <Button
-            outline
-            onClick={confirmDelete ? onCancel : onSave}
-            className="c-save-button">
-            {confirmDelete ? t('form.abort') : t('form.save')}
-          </Button>
-          <Button
-            outline
-            data-testid="warningModalConfirm"
-            onClick={onContinue}
-            className="c-save-button">
-            {confirmDelete
-              ? t('warningModal.delete')
-              : t('warningModal.continue')}
-          </Button>
+          {firstAction && (
```

Comment:
Something like `<ModalFooter actions={actions} component={footer} />` with a component override prop would be mor reusable. But we should refactor to Modal from ndla-ui soon anyway.